### DISCUSSION TITLE
Increase the number of egress and ingress pods based on requests in istio.

### DISCUSSION THREAD

**Discussion by gudipudipradeep** [istio/istio#49460] [Q&A]
Increase the number of egress and ingress pods based on requests in istio. how we can do that scale based on number of request.

**Comment by zirain** [comment]
istio provide metrics like `istio_request_*`, you can setup HPA base on those metrics.

**Reply by gudipudipradeep** [reply]
how to capture this istio_request_* in hpa deployment file. where we can see this metrics.

**Reply by gudipudipradeep** [reply]
need to install any per-requisites. like prometheus or inbuilt with istio.

**Reply by zirain** [reply]
that far beyond this project, https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-custom-metrics may help you.
